src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/ActivityTagsCollection.cs

```diff
@@ -0,0 +1,261 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Diagnostics
+{
+    /// <summary>
```

Comment:
nit: There are some typos/issues in this summary. Here's how I would edit if you want something to jump off of:
```csharp
	    /// <summary>
	    /// ActivityTagsCollection is a collection class used to store tracing tags.
	    /// This collection will be used with classes like <see cref="ActivityEvent"/> and <see cref="ActivityLink"/>.
	    /// This collection behaves as follows:
	    ///     - The collection items will be ordered according to how they are added.
	    ///     - Don't allow duplication of items with the same key.
	    ///     - When using the indexer to store an item in the collection:
	    ///         - If the item has a key that previously existed in the collection and the value is null, the collection item matching the key will be removed from the collection.
	    ///         - If the item has a key that previously existed in the collection and the value is not null, the new item value will replace the old value stored in the collection.
	    ///         - Otherwise, the item will be added to the collection.
	    ///     - Add method will add a new item to the collection if an item doesn't already exist with the same key. Otherwise, it will throw an exception.
	    /// </summary>
```

Reply:
Thanks for the writing.

Reply:
done. thanks.